File: length_of_last_word/length_of_last_word.py
class Solution:
    def lengthOfLastWord(self, s: str) -> int:
        s=s[::-1]
        s=s.lstrip(" ")
        count=0
        for i in range(0,len(s)):
            if(s[i]!=" "):
                count+=1
            else:
                break
        return count

File: 58_length_of_last_word/length_of_last_word.cpp
class Solution {
public:
    int lengthOfLastWord(string s) {
        int n =s.length();
        int count=0;
        int space=0;
        int flag=0;
        for (int i=n-1;i>=0;i--){
            if(s[i]==' ' && flag==0){
                space++;
            }
            else if(s[i]!=' '){
                flag=1;
            }
            else if(s[i]==' ' && flag==1){
                break;
            }
            count++;
        }
        return abs(space-count);
    }
};